source/microbit/help.c

```diff
@@ -81,58 +81,65 @@ STATIC const mp_doc_t help_table_types[] = {
-    {&microbit_pin_is_touched_obj, "is_touched() -- True if the pin is touched, otherwise False.\n"},
+    {&microbit_module, "Useful stuff to control the micro:bit hardware.\n"},
+    // System state objects
+    {&microbit_panic_obj, "Begin panic() mode (requires a reset).\n"},
+    {&microbit_random_obj, "Return a random(number) between 0 and 'number - 1'.\nrandom(10) returns a number between 0 and 9.\n"},
+    {&microbit_sleep_obj, "Put micro:bit to sleep(time) for some milliseconds (1 second = 1000 ms) of time.\nsleep(2000) gives micro:bit a 2 second nap.\n"},
+    {&microbit_running_time_obj, "Return running_time() in milliseconds since micro:bit's last reset.\n"},
+    // Accelerometer 3D orientation
+    {&microbit_accelerometer_obj, "Detect micro:bit's position (3D orientation).\n"},
+    {&microbit_accelerometer_get_x_obj, "Return micro:bit's X axis position.\n"},
+    {&microbit_accelerometer_get_y_obj, "Return micro:bit's Y axis position.\n"},
+    {&microbit_accelerometer_get_z_obj, "Return micro:bit's Z axis position.\n"},
+    // Pushbutton
+    {&microbit_button_a_obj, "micro:bit's 'A' button. When button is pressed down, is_pressed() is True.\n"},
+    {&microbit_button_b_obj, "micro:bit's 'B' button. When button is pressed down, is_pressed() is True.\n"},
+    {&microbit_button_is_pressed_obj, "If the button is pressed down, is_pressed() is True, else False.\n"},
+    // Compass 3D direction heading
+    {&microbit_compass_obj, "Use the compass to see what direction micro:bit is facing.\n"},
+    {&microbit_compass_is_calibrated_obj, "If micro:bit's compass is_calibrated() and adjusted for accuracy, return True.\nIf the compass hasn't been adusted for accuracy, return False.\n"},
+    {&microbit_compass_calibrate_obj, "Micro:bit, I command you to calibrate() and adjust your accuracy.\nWhere are my manners, please calibrate()"},
+    {&microbit_compass_is_calibrating_obj, "If micro:bit's compass is_calibrating() its accuracy, it sends True.\nIf it's not busy calibrating things, it sends False.\n"},
+    {&microbit_compass_clear_calibration_obj, "Reset micro:bit's compass using clear_calibration() command. Run calibrate() to improve accuracy.\n"},
+    {&microbit_compass_get_x_obj, "Return the compass' X axis direction.\n"},
+    {&microbit_compass_get_y_obj, "Return the compass' Y axis direction.\n"},
+    {&microbit_compass_get_z_obj, "Return the compass' Z axis direction.\n"},
+    // Display 5x5 LED grid
+    {&microbit_display_obj, "micro:bit's 5x5 LED display.\n"},
+    {&microbit_display_print_obj, "Use print(s) to print the string 's' to the display. Try print('Hello!').\nUse print(s, i) to print string 's', one character at a time with a delay of 'i' milliseconds.\n"},
+    {&microbit_display_scroll_obj, "Use scroll(s) to scroll the string 's' from left to right on the display.\nUse scroll(s, i) to scroll string 's' with a delay of 'i' milliseconds after each character.\n"},
```

Comment:
@carlosperate Probably would be best and more child friendly to remove the direction and mention ``character by character across the display``